mono/mini/debugger-engine.h

```diff
@@ -5,6 +5,9 @@
 #ifndef __MONO_DEBUGGER_ENGINE_H__
 #define __MONO_DEBUGGER_ENGINE_H__
 
+#include <mono/metadata/seq-points-data.h>
```

Comment:
Why do we need to include `seq-points-data.h`? If that's necessary only for `debugger-agent.c` then we prefer to put it there to reduce the need to rebuild everything whenever we modify one header file.

Reply:
The debugger-agent.c failed to compile for me for a while, master was broken. I think a subsequent fix was pushed. This was my workaround. I'll remove it. Me pushing this was an accident.